model.py

```diff
@@ -11579,15 +11579,24 @@ def works_updated_since(self, _db, timestamp):
         """Finds all works in a collection's catalog that have been updated
            since the timestamp. Used in the metadata wrangler.
 
-           :return: a Query
+           :return: a Query that yields (Work, LicensePool, Identifier,
+              WorkCoverageRecord, CollectionIdentifier) 5-tuples. With
+              the Work alone, the caller would not have the information
+              necessary to create full OPDS entries for the items.
```

Comment:
Without this, this branch will break the metadata wrangler. CatalogController.updates_feed will get a list of recently updated Works in a given Collection, but will have no way of knowing which Identifier associates the Work with the Collection. Previously this wasn't a problem because you didn't really need an Identifier to complete an OPDS entry, but now you do.

Reply:
Why do you need the CollectionIdentifier when you already know the collection?

Reply:
I don't need the CollectionIdentifier, but I couldn't find a way to use the CollectionIdentifier in the query but not return it in the results.

Reply:
You might be able to `.join` it separately instead of putting it in the first `query`.